packages/onchainkit/src/earn/components/RenderDepositButton.tsx

```diff
@@ -34,7 +34,7 @@ export function RenderDepositButton({
       </button>
     );
   }
-  if (context.errorMessage) {
+  if (context.errorMessage || depositAmountError) {
```

Comment:
i know it was like this before, but it seems strange that the success state is before the error state. is that because we don't clear errors on retries, so a user could have a `depositAmountError` and also a `context.receipt`?